packages/http/src/validator/validators/utils.ts

```diff
@@ -35,3 +39,6 @@ export const validateAgainstSchema = (value: any, schema: JSONSchema, prefix?: s
     throw new Error(`AJV validation error: "${error}"`);
   }
 };
+
+export const sequenceValidation = sequenceT(getValidation(getSemigroup<IPrismDiagnostic>()));
```

Comment:
In reality there are at least 3 new concepts here:

`sequenceT` and `sequenceS` — and then the `getValidation` that modifies the `ap` method of the `applicative` that the `Either` is implementing; from there, you then instruct the applicative how to concatenate instead of stopping by using a `semigroup`.

Do you want more? :trollface: 


> That line opened a whole new chapter of my FP learning.

We're just getting started.